### PR TITLE
Added PUT endpoint

### PR DESCRIPTION
**Pull request by dajwilson**

### CODE REVIEW

File: src/main/java/com/tasks/taskapi/controller/TaskController.java [hunk 1]
```diff
@@ -29,4 +30,11 @@ public ResponseEntity<Task> insertTask(Task task) {
         Task insertedTask = taskService.insertTask(task);
         return ResponseEntity.status(HttpStatus.OK).body(insertedTask);
     }
+
+    @PutMapping("/put")
```
Comment (bethanlawx) [inline src/main/java/com/tasks/taskapi/controller/TaskController.java]: i would use /edit or /update as its more user friendly - ntd tho

File: src/main/java/com/tasks/taskapi/controller/TaskController.java [hunk 2]
```diff
@@ -29,4 +30,11 @@ public ResponseEntity<Task> insertTask(Task task) {
...
+    public ResponseEntity<Task> editTask(Task task) {
+        System.out.println(task);
```
Comment (bethanlawx) [inline src/main/java/com/tasks/taskapi/controller/TaskController.java]: remove this <3

File: src/main/java/com/tasks/taskapi/repository/TaskRepository.java [hunk 3]
```diff
@@ -38,4 +43,10 @@ public int insertTask(Task task) {
 
         return (int) keyHolder.getKey();
     }
-}
+
+    public void editTask(Task task) {
+        String EDITED_EDIT_SQL = String.format(EDIT_SQL + WHERE_SQL, task.getName(),
```
Comment (bethanlawx) [inline src/main/java/com/tasks/taskapi/repository/TaskRepository.java]: formattedEditSql ? also would only use upper snake case at the top of the class but just convention

File: src/main/java/com/tasks/taskapi/controller/TaskController.java (hunk 1 above)
Comment (dajwilson) [reply src/main/java/com/tasks/taskapi/controller/TaskController.java]: so true

**Pull request merged**